eslint.config.js

```diff
@@ -84,15 +77,22 @@ export default [
       'react/no-multi-comp': ['error', { ignoreStateless: false }],
       'react/jsx-filename-extension': ['error', { extensions: ['.tsx'] }],
       'import/no-duplicates': 'error',
-      'tsdoc/syntax': 'error',
+      'no-undef': 'off',
       '@typescript-eslint/ban-ts-comment': 'error',
       '@typescript-eslint/no-unused-vars': 'error',
       '@typescript-eslint/no-explicit-any': 'error',
       '@typescript-eslint/no-non-null-assertion': 'error',
       '@typescript-eslint/consistent-type-assertions': 'error',
       '@typescript-eslint/naming-convention': [
         'error',
-        { selector: 'interface', format: ['PascalCase'], prefix: ['I'] },
+        {
```

Comment:
```suggestion
     {
  selector: 'interface',
  format: ['PascalCase'],
  custom: {
    regex: '^(?!I)[A-Z]',
    match: true,
  },
}   
```
Many in the TypeScript community including the core team discourage the use of "I" prefix.

Reply:
or better open a new issue and fix it for cleaner instead a blocker for this PR

Reply:
@varshith257  i had to add I as its widely used in the code base .

Reply:
@iamanishx Follow up this case with new issue

Reply:
@varshith257 can it be merge rn as i have my GSoC work so i think others can catch up from here

Reply:
okayy understand am opening a new issue to rename `I` contained interfaces in the codebase .
what will be the new rule ? will it be just Interface and TestInterface ?

Reply:
See above my suggestion